Makefile

```diff
@@ -221,7 +222,6 @@ image-registry: ## Build the testdata catalog used for e2e tests and push it to
 test-e2e: KIND_CLUSTER_NAME := operator-controller-e2e
 test-e2e: KUSTOMIZE_BUILD_DIR := config/overlays/e2e
 test-e2e: GO_BUILD_FLAGS := -cover
-test-e2e: CATALOGD_KUSTOMIZE_BUILD_DIR := catalogd/config/overlays/e2e
```

Comment:
👍 Now is all under  config/overlays/e2e cool

Reply:
Yes